tasks/app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<html>
```

Comment:
If this template is going to be rendered as a partial inside of the layout template in `tasks/app/views/layouts/application.html.erb` then we should not include all of the `html`, `head`, and `body` tags in here.
Since the only addition to the `head` in this template is the web font CSS link, I think it would make the most sense to add that into the layout directly.